### PR TITLE
Gamepagelayout

### PR DESCRIPTION
**Pull request by Hanttone**
Please review my code, it is mainly layout focused the design is currently only optimized for Iphone X/XS so keep in mind it can look a bit wonky with another preview. (I am also intending to change the pixel values to more scaling friendly units later on). Thanks already in advance for your time! :)

### CODE REVIEW

File: src/Services/GlobalStyle.js [hunk 1]
```diff
@@ -7,11 +7,22 @@ export default createGlobalStyle`
     padding: 0;
     margin: 0;
     font-size: 16px;
-    font-family: sans-serif;
+    font-family: Silom;
+    font-weight: regular;
+    color: white;
```
Comment (kschmitz123) [inline src/Services/GlobalStyle.js:12]: you could add global variables if you are using the same colors multiple times

**Review by kschmitz123 — APPROVED** [review]
lgtm 👍

File: src/Components/BottomLayout/BottomLayout.js [hunk 2]
```diff
@@ -1,67 +1,18 @@
-import { ReactComponent as Blue } from '../../Assets/blueWave.svg';
-import { ReactComponent as Lightgreen } from '../../Assets/greenFaded.svg';
-import { ReactComponent as Green } from '../../Assets/greenWave.svg';
-import { ReactComponent as Lightpurple } from '../../Assets/purpleFaded.svg';
-import { ReactComponent as Purple } from '../../Assets/purpleStrong.svg';
 import styled from 'styled-components/macro';
+import { ReactComponent as BottomDesign } from '../../Assets/bottomDesign.svg';
```
Comment (miriamhaenle) [inline src/Components/BottomLayout/BottomLayout.js:2]: I can't think of a better name but I have some issues with the name BottomDesign :D It is not a big issue but maybe you find a better name at one point. I'll also think about it.

File: src/Components/Lives/Lives.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components/macro';
+import { ReactComponent as Heart } from '../../Assets/heartGreen.svg';
+
+export default function Lives() {
+  return (
+    <LivesWrapper>
+      <Heart></Heart>
```
Comment (miriamhaenle) [inline src/Components/Lives/Lives.js:7]: Here you can use a void tag
```suggestion
      <Heart />
```

File: src/Components/Lives/Lives.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+```js
+<Lives></Lives>
```
Comment (miriamhaenle) [inline src/Components/Lives/Lives.md:2]: I think also here a void tag is sufficient

```suggestion
<Lives />
```

File: src/Components/NavItem/NavItem.js [hunk 5]
```diff
@@ -1,42 +1,54 @@
...
 
 export default function NavItem() {
-
-    /*<NavLinkStyled to="/settings" activeClassName="active">Settings</NavLinkStyled> */
-
-    return <NavWrapper>
-        <NavLinkStyled exact to="/" activeClassName="active">Home</NavLinkStyled>
-        <NavLinkStyled to="/game" activeClassName="active">Game</NavLinkStyled>
-        <NavLinkStyled to="/highscore" activeClassName="active">High Score</NavLinkStyled>
+  /*<NavLinkStyled to="/settings" activeClassName="active">Settings</NavLinkStyled> */
```
Comment (miriamhaenle) [inline src/Components/NavItem/NavItem.js:5]: No comments in main! :)

File: src/Components/NavItem/NavItem.js [hunk 6]
```diff
@@ -1,42 +1,54 @@
...
-grid-column: 2;
-grid-row: 1;
-z-index: 14;
-border-radius: 28px 0px 28px 28px;
-text-align: left;
-display: flex;
-flex-direction: column;
-gap: 5%;
-justify-content: center;
+  height: 299px;
```
Comment (miriamhaenle) [inline src/Components/NavItem/NavItem.js:30]: To make your CSS even nicer you could sort it alphabetically (or from the outside in).

File: src/Components/Scores/Scores.md [hunk 7]
```diff
@@ -0,0 +1,3 @@
+```js
+<Scores></Scores>
```
Comment (miriamhaenle) [inline src/Components/Scores/Scores.md:2]:
```suggestion
<Scores />
```

File: src/Components/TimerBar/TimerBar.js [hunk 8]
```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components/macro';
+import { ReactComponent as Timer } from '../../Assets/timer.svg';
+
+export default function TimerBar() {
+  return (
+    <TimerWrapper>
+      <Timer></Timer>
```
Comment (miriamhaenle) [inline src/Components/TimerBar/TimerBar.js:7]:
```suggestion
      <Timer />
```

File: src/Components/WordCard/WordCard.js [hunk 9]
```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components';
+
+export default function WordCard({ word }) {
+  return (
+    <WordCardWrapper>
+      <div></div>
```
Comment (miriamhaenle) [inline src/Components/WordCard/WordCard.js:6]: Is the div here only for styling purposes?
Use a void tag then :) <div />.

File: src/Components/WordCard/WordCard.md [hunk 10]
```diff
@@ -0,0 +1,3 @@
+```js
+<WordCard></WordCard>
```
Comment (miriamhaenle) [inline src/Components/WordCard/WordCard.md:2]:
```suggestion
<WordCard />
```

File: src/Pages/Game.js [hunk 11]
```diff
@@ -4,26 +4,38 @@ import Navigation from '../Components/Navigation/Navigation';
...
 import Button from '../Components/Button/Button';
+import WordCard from '../Components/WordCard/WordCard';
+import Scores from '../Components/Scores/Scores';
+import Lives from '../Components/Lives/Lives';
+import TimerBar from '../Components/TimerBar/TimerBar';
 
 export default function GamePage() {
   return (
     <GameWrapper>
       <Navigation></Navigation>
```
Comment (miriamhaenle) [inline src/Pages/Game.js:15]:
```suggestion
      <Navigation />
```

File: src/Pages/Game.js [hunk 12]
```diff
@@ -4,26 +4,38 @@ import Navigation from '../Components/Navigation/Navigation';
...
+          text="Die"
+          zindex="1"></Button>
         <Button
           width="100%"
           radius="20px"
           text="Das"
-          row="5"></Button>
+          zindex="1"></Button>
       </ButtonWrapper>
       <BottomLayout></BottomLayout>
```
Comment (miriamhaenle) [inline src/Pages/Game.js:40]:
```suggestion
      <BottomLayout />
```

File: src/Pages/Home.js [hunk 13]
```diff
@@ -8,12 +8,13 @@ export default function Home() {
...
-        text="Play"
-        row="4"></Button>
+      <Header>Der, die & das</Header>
+      <ButtonWrapper>
+        <Button
+          width="80%"
+          radius="28px"
+          text="Play"></Button>
+      </ButtonWrapper>
       <BottomLayout></BottomLayout>
```
Comment (miriamhaenle) [inline src/Pages/Home.js:18]:
```suggestion
      <BottomLayout />
```

File: src/Services/GlobalStyle.js [hunk 14]
```diff
@@ -1,4 +1,4 @@
-import { createGlobalStyle } from "styled-components";
+import { createGlobalStyle } from 'styled-components';
```
Comment (miriamhaenle) [inline src/Services/GlobalStyle.js:1]:
```suggestion
import { createGlobalStyle } from 'styled-components/macro';
```

**Review by miriamhaenle — COMMENTED** [review]
Nicely done 👏 

I did not comment on all your px Values as you wrote that you intend to change that later on. Otherwise I would have commented to change them to rem/em

File: src/Components/NavItem/NavItem.js (hunk 5 above)
Comment (Hanttone) [reply src/Components/NavItem/NavItem.js:5]: This is just there to save the code for later once I am implementing the settings page. :D

File: src/Services/GlobalStyle.js (hunk 1 above)
Comment (Hanttone) [reply src/Services/GlobalStyle.js:12]: Thanks for the idea! :)

**Pull request merged**